tests/testthat/test-ppc-distributions.R

```diff
@@ -58,6 +58,22 @@ test_that("ppc_dens,pp_hist,ppc_freqpoly,ppc_boxplot return ggplot objects", {
   expect_gg(ppd_boxplot(yrep2, notch = FALSE))
 })
 
+test_that("ppc_qdotplot returns a ggplot object", {
+  testthat::skip_if_not_installed("ggdist")
+
+  expect_gg(ppc_qdotplot(y, yrep[1:8, ]))
+  expect_gg(ppc_qdotplot(y, yrep[1,, drop = FALSE], quantiles=25))
+  expect_gg(ppc_qdotplot(y, yrep[1:8, ], binwidth = 0.1))
+  expect_gg(ppc_qdotplot(y2, yrep2, binwidth = 0.1, quantiles = 25))
+
+
+  # ppd versions
+  expect_gg(ppd_qdotplot(y, yrep[1:8, ]))
+  expect_gg(ppd_qdotplot(yrep[1,, drop = FALSE], quantiles = 25))
+  expect_gg(ppd_qdotplot(yrep[1:8, ], binwidth = 0.1))
+  expect_gg(ppd_qdotplot(yrep2, binwidth = 0.1, quantiles = 25))
+})
+
```

Comment:
If some (or all) of these will have warning messages about overflow, can you wrap the relevant ones in `expect_warning()`?